src/main/java/staffconnect/logic/parser/AddCommandParser.java

```diff
@@ -48,8 +51,10 @@ public AddCommand parse(String args) throws ParseException {
         Venue venue = ParserUtil.parseVenue(argMultimap.getValue(PREFIX_VENUE).get());
         Module module = ParserUtil.parseModule(argMultimap.getValue(PREFIX_MODULE).get());
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
+        Set<Availability> availabilityList =
+                ParserUtil.parseAvailabilities(argMultimap.getAllValues(PREFIX_AVAILABILITY));
```

Comment:
Default behaviour if the the user types nothing, the availability is none of the days, rather than any of the days

Reply:
I think that behaviour is intuitive imo, as opposed to the alternative be that availability be defaulted to all the days